test/Microsoft.AspNetCore.Razor.Evolution.Test/IntegrationTests/RazorIRNodeWriter.cs

```diff
@@ -0,0 +1,162 @@
+        {
+            WriteContentNode(node, node.AccessModifier, node.Name, node.BaseType, string.Join(", ", node.Interfaces ?? new List<string>()));
+        }
+
+        public override void VisitCSharpAttributeValue(CSharpAttributeValueIRNode node)
+        {
+            WriteContentNode(node, node.Prefix, node.Content.ToString()); // This is broken.
+        }
+
+        public override void VisitCSharpExpression(CSharpExpressionIRNode node)
+        {
+            WriteContentNode(node, node.Content.ToString()); // This is broken
+        }
+
+        public override void VisitCSharpStatement(CSharpStatementIRNode node)
+        {
+            WriteContentNode(node, node.Content);
+        }
+
+        public override void VisitCSharpToken(CSharpTokenIRNode node)
+        {
+            WriteContentNode(node, node.Content);
+        }
+
+        public override void VisitDirective(DirectiveIRNode node)
+        {
+            WriteContentNode(node, node.Name);
+        }
+
+        public override void VisitDirectiveToken(DirectiveTokenIRNode node)
+        {
+            WriteContentNode(node, node.Content);
+        }
+
+        public override void VisitHtml(HtmlContentIRNode node)
+        {
+            WriteContentNode(node, node.Content);
+        }
+
+        public override void VisitHtmlAttribute(HtmlAttributeIRNode node)
+        {
+            WriteContentNode(node, node.Prefix, node.Value.ToString(), node.Suffix);
+        }
+
+        public override void VisitHtmlAttributeValue(HtmlAttributeValueIRNode node)
+        {
+            WriteContentNode(node, node.Prefix, node.Content);
+        }
+
+        public override void VisitNamespace(NamespaceDeclarationIRNode node)
+        {
+            WriteContentNode(node, node.Content);
+        }
+
+        public override void VisitRazorMethodDeclaration(RazorMethodDeclarationIRNode node)
+        {
+            WriteContentNode(node, node.AccessModifier, string.Join(", ", node.Modifiers ?? new List<string>()), node.ReturnType, node.Name);
+        }
+
+        public override void VisitUsingStatement(UsingStatementIRNode node)
+        {
+            WriteContentNode(node, node.Content);
+        }
+
+        protected void WriteBasicNode(RazorIRNode node)
+        {
+            WriteIndent();
+            WriteName(node);
+            WriteSeparator();
+            WriteLocation(node);
+        }
+
+        protected void WriteContentNode(RazorIRNode node, params string[] content)
+        {
+            WriteIndent();
+            WriteName(node);
+            WriteSeparator();
+            WriteLocation(node);
+
+            for (var i = 0; i < content.Length; i++)
+            {
+                WriteSeparator();
+                WriteContent(content[i]);
+            }
+        }
+
+        protected void WriteIndent()
+        {
+            for (var i = 0; i < Depth; i++)
+            {
+                _writer.Write('\t');
```

Comment:
Since most of us have tabs => spaces can we write out spaces here instead of tabs in case we open a baseline in an editor. Will help us avoid accidentally modifying baselines.